spec/rubocop/cop/rspec/factory_bot/attribute_defined_statically_spec.rb

```diff
@@ -1,22 +1,57 @@
 # frozen_string_literal: true
 
 RSpec.describe RuboCop::Cop::RSpec::FactoryBot::AttributeDefinedStatically do
+  def inspected_source_filename
```

Comment:
Our overloaded `expect_offense`/`expect_no_offenses` use this to pass the file name to `super`.

This tells the cop that it inspects a relevant source.